src/ansiblelint/rules/IgnoreErrorsRule.py

```diff
@@ -0,0 +1,92 @@
+"""IgnoreErrorsRule used with ansible-lint."""
+import sys
+from typing import TYPE_CHECKING, Any, Dict, Union
+
+from ansiblelint.rules import AnsibleLintRule
+
+if TYPE_CHECKING:
+    from typing import Optional
+
+    from ansiblelint.file_utils import Lintable
+
+
+class IgnoreErrorsRule(AnsibleLintRule):
+    """Describe and test the IgnoreErrorsRule."""
+
+    id = "ignore-errors"
+    shortdesc = 'Use failed_when and specify error codes instead of ignore_errors'
+    description = (
+        'Instead of ignoring all errors, use ``failed_when:`` '
+        'and specify acceptable error codes '
+        'to reduce the risk of ignoring important failures'
+    )
+    severity = 'LOW'
+    tags = ['unpredictability', 'experimental']
+    version_added = 'v5.0.7'
+
+    def matchtask(
+        self, task: Dict[str, Any], file: 'Optional[Lintable]' = None
+    ) -> Union[bool, str]:
+
+        if task.get("ignore_errors"):
+            return True
+
+        return False
+
+
+if "pytest" in sys.modules:
+    import pytest
+
+    IGNORE_ERRORS_TRUE = '''
+- hosts: all
+  tasks:
+    - name: run apt-get update
+      command: apt-get update
+      ignore_errors: true
+'''
+
+    IGNORE_ERRORS_FALSE = '''
+- hosts: all
+  tasks:
+    - name: run apt-get update
+      command: apt-get update
+      ignore_errors: false
```

Comment:
Maybe this should be also its own rule? After all, `ignore_errors: false` is a quite nonsensical statement and might be a hint for some change gone wrong that was intended to be made elsewhere.

Reply:
I agree, it was mostly added as a test case as hinted on https://github.com/ansible-community/ansible-lint/pull/1540/files#diff-8d1811f448323a979183ecf2d278ac0956ac3b60f91cc22eeb1c777b24520537R82.